package.json

```
@@ -2,7 +2,7 @@
   "author": "Transcend Inc.",
   "name": "@transcend-io/privacy-types",
   "description": "Core enums and types that can be useful when interacting with Transcend's public APIs.",
-  "version": "4.13.0",
+  "version": "4.13.1",
```

```suggestion
  "version": "4.14.0",
```

nit: i would consider a new constant a "new backwards compatible feature" in a repo that just holds constants & types, so i'd probably bump the minor version instead of the patch